File: pgpainless-core/src/main/java/org/pgpainless/exception/WrongConsumingMethodException.java
// SPDX-FileCopyrightText: 2021 Paul Schaub <vanitasvitae@fsfe.org>
//
// SPDX-License-Identifier: Apache-2.0

package org.pgpainless.exception;

import org.bouncycastle.openpgp.PGPException;

public class WrongConsumingMethodException extends PGPException {

    public WrongConsumingMethodException(String message) {
        super(message);
    }
}

File: pgpainless-core/src/main/kotlin/org/pgpainless/exception/WrongConsumingMethodException.kt
// SPDX-FileCopyrightText: 2025 Paul Schaub <vanitasvitae@fsfe.org>
//
// SPDX-License-Identifier: Apache-2.0

package org.pgpainless.exception

import org.bouncycastle.openpgp.PGPException

class WrongConsumingMethodException(message: String) : PGPException(message)
